src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,19 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int FROM_YEAR_INDEX = 0;
+    private static final int TO_YEAR_INDEX = 1;
+
+    @Override
+    public boolean test(Candidate candidate) {
+        String[] splitedYears = candidate.getPeriodsInUkr().split("-");
+        return candidate.isAllowedToVote()
+                && candidate.getAge() >= 35
```

Comment:
```suggestion
                && candidate.getAge() >= MIN_AGE
```
Use constant instead magic numbers